docs/install/operator/knative-with-operators.md

```diff
@@ -49,6 +49,13 @@ kubectl logs -f deploy/knative-operator
 
 ## Installing the Knative Serving component
 
+### Prepare the networking layer
+
+Knative Serving relies on the networking layer to manage the ingress and egress traffic. Istio is the default network layer
+for Knative Operator configuration. Please follow [Install Istio](../serving/installing-istio.md) before proceeding.
+
+If you would like to leverage networking layers other than Istio, directly go to [this section](#installing-knative-serving-with-different-networking-layers).
```

Comment:
I have a few questions about this new section:

1. Do users have to install the networking layer before applying the Serving CR? Or do they just have to install Istio at this point if they want to use the default networking layer. 
2. If they want to use other networking layers, should they install and configure them now?
3. I find it a bit confusing that there are two sections for installing the networking layer. Would it make sense to move the section "Installing Serving with different networking layers" before the section for applying the Serving CR instead of having this short section that links to another one below? 
That way users can do the sections in the order they appear on the page instead of skipping down to another section and then having to go back up to apply the Serving CR.

cc: @houshengbo